docs/WebSocket.md

```diff
@@ -59,9 +83,9 @@ The number of bytes of data that have been buffered (queued but not yet transmit
 
 ##### Instances
 ``` purescript
-Generic BufferedAmount
-Eq BufferedAmount
-Ord BufferedAmount
+instance genericBufferedAmount :: Generic BufferedAmount
+instance eqBufferedAmount :: Eq BufferedAmount
+instance ordBufferedAmount :: Ord BufferedAmount
```

Comment:
Could you regenerate the docs using the latest release of `psc` so that this changes don't show up. Starting from [0.7.6](https://github.com/purescript/purescript/releases/tag/v0.7.6) `psc-docs` doesn't produce these noisy `instance ... :: ` lines.